Makefile

```diff
@@ -247,6 +247,28 @@ check:
 	source ${PROTEUS_PREFIX}/bin/proteus_env.sh; mpirun -np 4 ${PROTEUS_PYTHON} proteus/tests/ci/test_meshPartitionFromTetgenFiles.py
 	@echo "************************"
 
+	@echo "************************"
+	@echo "SCOREC-Proteus Test"
+	${PROTEUS_ENV} ${PROTEUS_PYTHON} proteus/MeshAdaptPUMI/test/meshLoad/meshLoad.py
+	@echo "************************"
+  
+	@echo "SCOREC-Gmsh Flow Test"
+	${PROTEUS_ENV} ${PROTEUS_PYTHON} proteus/MeshAdaptPUMI/test/test_gmshLoadAndAdapt/gmshCheck.py
+	@echo "************************"
+
```

Comment:
Agreed. I added an additional build target check_simmetrix in the latest commit.